app/views/govuk_publishing_components/components/_layout_header.html.erb

```diff
@@ -0,0 +1,61 @@
+<%
+navigation_classes ||= ''
+  navigation_items ||= []
+  css_classes ||= ''
+  container_classes ||= ''
+  homepage_url ||= '/'
+  assets_path ||= '/assets/images'
+  product_name ||= nil
```

Comment:
What's the difference between `product_name` and `service_name`?

Reply:
Product name is the one displayed next to GOV.UK on the lest side (e.g. https://www.payments.service.gov.uk/), the product name is mainly used for services and is displayed on the two-thirds container (e.g. https://www.registertovote.service.gov.uk/register-to-vote/country-of-residence)

Reply:
Ah ok thanks! Maybe we should start just with supporting product name? I think we'll go with that for the admin apps.